src/client/components/ProgressButton.jsx

```diff
@@ -22,10 +24,12 @@ class ProgressButton extends React.Component {
     const afterText = (this.props.afterText) ? (<span className="button-icon"> {this.props.afterText}</span>) : '';
 
     return (
-      <button
-          className={`text-capitalize ${this.props.buttonClass}`}
-          id={`${this.id}-continueButton`}
-          onClick={this.props.onButtonClick}>{beforeText}{this.props.buttonText}{afterText}</button>
+      <span className={this.props.additionalClass}>
```

Comment:
I think these changes can be reverted too if we're removing the `additionalClass` prop on the buttons.